randovania/games/dread/logic_database/Artaria.txt

```diff
@@ -758,6 +758,8 @@ Extra - asset_id: collision_camera_008
       Trivial
   > Door to EMMI First Chase End (Top)
       Trivial
+  > Lower Room
+      Before Artaria - Thermal Device and Shinesink Clip (Intermediate) and Can SSC
```

Comment:
Why does activating the thermal device remove your ability to move between these nodes?

Reply:
Yeah, not sure why that's there. that thermal door has no bearing on being able to charge speed.

Reply:
this is because you need to have a wall to sink off of. the other door is a door frame so you couldn't if the door is opened. 

though now that I think about it, it shouldn't logically matter. if the thermal device is open you can access the room to the left, and if you would be expected to sink and go to the right exits, either the right door is open and you can just  spark up the wall or it's a locked door and you can sink off that.